templates/admin/polls/index.html

```diff
@@ -0,0 +1,14 @@
+<% include adminheader.ejs %>
+<script>
+function createPoll() {
+	socket.emit('createPoll')
+	window.location.href = "/admin/poll/create"
+}
+</script>
+<div class="top"><h1>Polls</h1></div>
+<div class="search"><input id="searchBar" type="text"></div>
+<div class="main">
+<button onclick="">Create poll</button>
```

Comment:
If you have any questions, just let me know! It might be a little advanced, but it might be something interesting to look into. Continuing like this, might teach you to use bad practices.